docs/security/datasource_authorization.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
-title: "Data Source Authorization"
-description: "Data Source Authorization"
+title: "Data Source Authorization in Apache Zeppelin"
+description: "Data source authorization involves authenticating to the data source like a Mysql database and letting it determine user permissions. Apache Zeppelin allows users to use their own credentials to authenticate with Data Sources."
```

Comment:
Apache Zeppelin supports protected data sources. I.e in case of a MySql database - every user can set up his own credentials to access it.